test/functional/forall/test-forall-atomic-utils.hpp

```diff
@@ -0,0 +1,80 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef __TEST_FORALL_ATOMIC_UTILS_HPP__
+#define __TEST_FORALL_ATOMIC_UTILS_HPP__
+
+#include "RAJA/RAJA.hpp"
+
+#include "test-forall-utils.hpp"
+
+using AtomicSeqExecs = camp::list< RAJA::seq_exec,
+                                   RAJA::loop_exec
+                                   //RAJA::simd_exec  // fails for atomic-view with clang9
```

Comment:
Should we expect atomics and simd to work together?

Reply:
No. They do not. @rchen20 this is a case where the policy list must be different for forall and forall-atomic tests.